src/atoms/Identicon/Identicon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import styled from '_styled-components';
+import makeBlockie from 'ethereum-blockies-base64';
+
+const RoundedImage = styled.img`
+  border-radius: 100%; /* stylelint-disable-line unit-whitelist */
+`;
```

Comment:
Looks good, but maybe we should make it smaller to make it match the address design (or whatever size is most common).

Reply:
Nevermind, I'm fine with keeping this size, I'll just resize it when I implement Address.